package.json

```
@@ -6,12 +6,15 @@
     "build-all": "yarn run build-ui && yarn run build-css",
     "postinstall": "cd css && npm i",
     "clean": "rm -rf output .spago node_modules && cd css && rm -rf node_modules",
-    "build-ui": "yarn run spago build -p \"ui-guide/**/*.purs\" -u \"$RTS_ARGS\" && purs bundle output/*/*.js -o dist/example.js $RTS_ARGS",
-    "watch-ui": "yarn run spago bundle-app -w -u \"$RTS_ARGS\" --then 'yarn run build-css' -p \"ui-guide/**/*.purs\" --to dist/example.js",
+    "build-ui": "yarn run build-ui-build && yarn run build-ui-bundle && yarn run build-ui-browserify",
+    "build-ui-build": "yarn run spago build -p \"ui-guide/**/*.purs\" -u \"$RTS_ARGS\"",
+    "build-ui-bundle": "purs bundle output/*/*.js --main Main --module Main -o dist/bundled.js $RTS_ARGS",
+    "build-ui-browserify": "yarn run browserify dist/main.js --outfile dist/index.js",
```

I we're using the wrong filename here. Its causing the build-ui step to fail. 
Either change this to `bundled.js` or change the bundler to use `main.js`
```suggestion
    "build-ui-browserify": "yarn run browserify dist/bundled.js --outfile dist/index.js",
```

---

That wasn't a typo actually. As mentioned in the latest commit, we meant to add a new `dist/main.js` in the first commit but the `dist/` directory is in `.gitignore` so we missed it.